### DISCUSSION TITLE
gpg import fails awaiting for prompt answer

### DISCUSSION THREAD

**Discussion by foxhound91** [renovatebot/renovate#16304] [Suggest an Idea]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.97.0

### Please select which platform you are using if self-hosting.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

14.8.2-ee

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

If setting the [gitPrivateKey](https://docs.renovatebot.com/self-hosted-configuration/#gitprivatekey) config to enable GPG signing this fails in some cases because the import process prompts the user for the password for that key.
To fix this we should add by default the `--batch` flag to the `gpg import` command.

### Relevant debug logs

<details><summary>Logs</summary>

```
Error building skey array: No such file or directory
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
Converted to discussion, as it's not a bug

![image](https://user-images.githubusercontent.com/1798109/176173806-86201b37-cc10-4ffe-9c2f-f7bd61a0d51d.png)
https://docs.renovatebot.com/self-hosted-configuration/#gitprivatekey

Even adding `--batch`  would fail, as renovate doesn't support protected keys and the docs also mention that.

Feel free to open a feature request to add `--batch` to import command.
